packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -516,3 +516,7 @@ export function amplifyStatus(cwd: string, expectedStatus: string, testingWithLa
 export function initHeadless(cwd: string, envName: string, appId: string): Promise<void> {
   return spawn(getCLIPath(), ['init', '--yes', '--envName', envName, '--appId', appId], { cwd, stripColors: true }).runAsync();
 }
+
+export function initNonInteractive(cwd: string): Promise<void> {
```

Comment:
can we just re-use the helper function ^? You should be able to just make the 2nd & 3rd parameter optional.

Reply:
this is good feedback. let's do this.

Reply:
addressed this in the latest commit